config.js

```diff
@@ -19,7 +19,8 @@ const conf = {
     cert: process.env.LEV_TLS_CERT || null,
     ca: process.env.LEV_TLS_CA || null,
     verify: String(process.env.LEV_TLS_VERIFY).match(/false/i) === null
-  }
+  },
+  MAX_AUDIT_RANGE: 92
```

Comment:
Why the caps? Because it is a temporary measure?

Reply:
No kind of the opposite, it's constant (albeit set by the config), it's a kind of Java-ism

Reply:
...Java. :-/